tests/unit/worker/test_submission_worker.py

```diff
@@ -79,7 +168,51 @@ def test_create_dir_as_python_package(self):
 
     def test_return_file_url_per_environment(self):
         returned_url = return_file_url_per_environment(self.url)
-        self.assertEqual(returned_url, "http://testserver/test/url")
+        self.assertEqual(returned_url, "{0}{1}".format(self.testserver, self.url))
+
+    @mock.patch("scripts.workers.submission_worker.create_dir_as_python_package")
+    @mock.patch("scripts.workers.submission_worker.download_and_extract_file")
+    def test_extract_submission_data_success(
+            self,
+            mock_download_and_extract_file,
+            mock_create_dir_as_python_package):
+        submission_input_file_path = join(
```

Comment:
As I can see that we only use `SUBMISSION_DATA_DIR` with `{input_file}` so can we define this directly in it's declaration. As this line is redundant to line 139

Reply:
We need it to mock `scripts.worker.submission_worker.SUBMISSION_DATA_DIR`, I think you missed that it has been used on [line 182](https://github.com/Cloud-CV/EvalAI/pull/2498/files#diff-1aeba0ea1ecc134c604f070c6be1385aR182). I think we should keep it as it is.